extensions/objects/models/objects.py

```diff
@@ -530,8 +530,8 @@ def normalize(cls, raw):
                             expression.
                         top_operator_name: str. The top operator name
                             in the expression.
-                        arguments: list. A list of arguments.
-                        dummies: list. A list of dummy values.
+                        arguments: list(str). A list of arguments.
+                        dummies: list(double). A list of dummy values.
```

Comment:
Python does not have "double" as a type. Please provide the correct type and confirm how you figured it out.

Reply:
Could never find a case where it was used in the code base where I could print out the type since all of the tests had dummies as empty. Will just leave it as is then, arguments however, I did find examples of arguments in tests.